### DISCUSSION TITLE
How to share the network management between 2 or more users

### DISCUSSION THREAD

**Discussion by rogertgu** [sinamics/ztnet#169] [Q&A] [answered]
Hi team, 

First of all, I would like to express my amazement with this wonderful interface that I personally find much better and more modern than the official zerotier-central dashboard, @sinamics has done a great job here!

I was wondering if there is a possibility to share the network management of a specific network between multiple users, 
this could be useful if for example I share my network with my brother and I want to let him to allow new devices to connect to a "shared" network and let him change the ip of the devices and other options of that specific network. 

I've been thinking that I could achieve this if i create a new user on zerotier-central and share that api secret, so we could manage the  networks created there, but i'm not sure if this is possible in an easiest way. 

Thanks in advance!

**Comment by sinamics** [comment, marked as answer]
Hi
Currently this is not available, but im working on a branch that will enable organization and i think this is what you are looking for.
With Organization, users can create and manage networks collectively, allowing for shared administration privileges over the specified networks. 
You and your brother could be part of the same Organization, and thereby have the ability to add new devices, change IP addresses, and modify other network settings collaboratively. 

https://github.com/sinamics/ztnet/pull/99

**Comment by PhaNtomBek** [comment]
Greetings. I'm looking for a UI for ZeroTier. I really liked your creation. But I need the same function as @rogertgu  asked. May I know when you plan to implement this solution approximately?

**Reply by sinamics** [reply]
This will probably not be completed before Q1 next year. Its quite a comprehensive task and there is only 24h a day.

**Comment by sinamics** [comment]
@rogertgu @PhaNtomBek 
https://github.com/sinamics/ztnet/discussions/221

**Comment by Prikith** [comment]
Would it be possible to add a feature to move user networks into an organisation and between organisations? and possibly adding a zt central key to an organisation to allow for easy management of backup networks per organisation

**Reply by sinamics** [reply]
I can add a feature to move private networks to an organization, but I don't want to involve zerotier cental into the organization segment.

**Reply by sinamics** [reply]
https://github.com/sinamics/ztnet/releases/tag/0.5.4

**Reply by Prikith** [reply]
Thanks for the quick solution. For the future it might be nice to have the roles admin and superadmin in an organization where a superadmin can't kick other superadmins but regular admins can still perform highly privileged tasks like inviting and kicking users. This would work with my next suggestion being able to move a network from one organization to another organization if you have the role superadmin in the organization where the network is currently located. Obviously this comes with extra headaches like not allowing you to delete an organization if there are multiple superadmins. Another naming scheme could be moderator and admin but that's just preference.
